redis/client.py

```diff
@@ -2996,12 +2996,23 @@ def hlen(self, name):
         "Return the number of elements in hash ``name``"
         return self.execute_command('HLEN', name)
 
-    def hset(self, name, key, value):
+    def hset(self, name, key=None, value=None, mapping=None):
         """
-        Set ``key`` to ``value`` within hash ``name``
-        Returns 1 if HSET created a new field, otherwise 0
+        Set ``key`` to ``value`` within hash ``name``,
+        Use ``mappings`` keyword args to set multiple key/value pairs
```

Comment:
Small typo here: `s/mappings/mapping`

Reply:
Thanks, but it's already fixed in master: https://github.com/andymccurdy/redis-py/blob/master/redis/client.py#L3037